src/usr/local/www/system_advanced_network.php

```diff
@@ -158,23 +158,25 @@
 ))->setHelp('NOTE: This does not disable any IPv6 features on the firewall, it only '.
 	'blocks traffic.');
 
+
 $group = new Form_Group('IPv6 over IPv4 Tunneling');
 $group->add(new Form_Checkbox(
 	'ipv6nat_enable',
 	'IPv6 over IPv4 Tunneling',
-	'Enable IPv4 NAT encapsulation of IPv6 packets',
+	'Enable IPv6 over IPv4 tunneling',
```

Comment:
Then make the left-side label "IPv6 over IPv4" and the checkbox label "Enable IPv6 over IPv4 Tunneling". I'd still prefer to see the longer checkbox label since it reads much less redundantly. The other text is far enough away the repetition isn't bad, whereas here it's 3x in a row (the group title, option label, and checkbox label).